src/periphery/StEthOracle.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.18;
+
+import {IStEth} from "../interfaces/IStEth.sol";
+
+import {BaseOracle} from "./BaseOracle.sol";
+
+contract StEthOracle is BaseOracle {
```

Comment:
this is the WstEthOracle

which is correct given the pool we're targetting https://curve.fi/#/ethereum/pools/factory-twocrypto-44/deposit

but name of oracle needs changing